nin.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "My project",
+  "authors": [
+    "Your demoscene handle"
+  ],
+  "description": "This is my project",
+  "previewImage": "https://raw.githubusercontent.com/ninjadev/nin/master/nin/frontend/app/images/nin-dark.png",
+  "music": {
+    "path": "res/music.mp3",
+    "bpm": 125,
+    "subdivision": 4
+  },
+  "googleAnalyticsID": "Your GA ID"
```

Comment:
We could set this to empty string instead, and have `nin compile` complain if it is unset, so it's harder to forget.